.github/workflows/test.yml

```diff
@@ -16,6 +16,7 @@ concurrency:
 jobs:
   integration_tests:
     runs-on: ubuntu-latest
+    timeout-minutes: 180
```

Comment:
mmm it is still quite long?

Maybe the default timeout for wait_for_fn could also be reduce

Reply:
yes, for case like go [update](https://github.com/crypto-org-chain/cronos/pull/845)

Reply:
the log seems not wait but just workflow stuck
```
2023-02-09T03:11:47.7203425Z test_basic.py::test_tx_inclusion[True-25000000] 2023-02-09 03:11:47,719 INFO waiting for cronos_777-1-node0 to stop
2023-02-09T03:11:47.7229433Z 2023-02-09 03:11:47,722 INFO stopped: cronos_777-1-node0 (terminated by SIGTERM)
2023-02-09T03:11:48.2776025Z 2023-02-09 03:11:48,277 INFO spawned: 'cronos_777-1-node0' with pid 24606
2023-02-09T03:11:48.2838310Z 2023-02-09 03:11:48,283 INFO stopped: cronos_777-1-node1 (terminated by SIGTERM)
2023-02-09T03:11:48.2877096Z 2023-02-09 03:11:48,287 INFO spawned: 'cronos_777-1-node1' with pid 24607
2023-02-09T03:11:48.2917123Z cronos_777-1-node0: stopped
2023-02-09T03:11:48.2917811Z cronos_777-1-node0: updated process group
2023-02-09T03:11:48.3055928Z cronos_777-1-node1: stopped
2023-02-09T03:11:48.3056308Z cronos_777-1-node1: updated process group
2023-02-09T03:11:48.3056606Z cronos_777-1-node2: stopped
2023-02-09T03:11:48.3056901Z cronos_777-1-node2: updated process group
2023-02-09T03:11:51.7649237Z 2023-02-09 03:11:51,764 INFO success: cronos_777-1-node0 entered RUNNING state, process has stayed up for > than 3 seconds (startsecs)
2023-02-09T03:11:51.7650298Z 2023-02-09 03:11:51,764 INFO success: cronos_777-1-node1 entered RUNNING state, process has stayed up for > than 3 seconds (startsecs)
2023-02-09T08:12:31.5353765Z ##[error]The operation was canceled.
```

Reply:
I see, then should be ok.

I have seen a lot of random failure on the integration tests lately. May be worth to investigate the reason